src/components/Main/Footer/Contact.tsx

```diff
@@ -1,23 +1,45 @@
 import React from 'react';
+import { useState } from 'react';
 import discordSocial from '../../../assets/images/discord-icon.svg';
 import instagramSocial from '../../../assets/images/instagram-icon.svg';
 import facebookSocial from '../../../assets/images/facebook-icon.svg';
 import twitterSocial from '../../../assets/images/twitter-icon.svg';
+import discordSocialBlue from '../../../assets/images/discord-icon-blue.svg';
+import instagramSocialBlue from '../../../assets/images/instagram-icon-blue.svg';
+import facebookSocialBlue from '../../../assets/images/facebook-icon-blue.svg';
+import twitterSocialBlue from '../../../assets/images/twitter-icon-blue.svg';
 import {isMobile} from 'react-device-detect';
 
 function Contact() {
+  const [hovered, setHovered] = useState(false);
+  const toggleHover = () => setHovered(!hovered);
   return (
     <div>
       <h3 className="pb-2"><strong>Contact</strong></h3>
-      <a href="mailto:gwcuva@gmail.com" className="mono text-white interactive">gwcuva@gmail.com</a>
+      <a href="mailto:gwcuva@gmail.com" className={hovered ?  "mono text-peacock interactive" : "mono text-white interactive"}
```

Comment:
why not edit the css file so interactive:hover is peacock instead of white? this helps in case we ever decide to switch up the color